lib/instance.js

```diff
@@ -1024,7 +1025,7 @@ module.exports = (function() {
         delete defaults[this.Model._timestampAttributes.deletedAt];
       }
 
-      if (Object.keys(defaults).length) {
+      if (Object.keys(defaults).length !== 0) {
```

Comment:
I prefer the explicit one check. I'm uncertain if V8 handles it faster but this would not be measurable if. So it's just the style^^